metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorFactory.java

```diff
@@ -75,17 +79,17 @@ public HiveConnectorFactory(final String name, final Map<String, String> configu
     private void init() {
         try {
             final boolean useLocalMetastore = Boolean
-                .parseBoolean(configuration.getOrDefault(USE_EMBEDDED_METASTORE, "false"));
+                    .parseBoolean(configuration.getOrDefault(USE_EMBEDDED_METASTORE, "false"));
             if (!useLocalMetastore) {
                 client = createThriftClient();
             } else {
                 client = createLocalClient();
             }
         } catch (Exception e) {
             throw new IllegalArgumentException(
-                String.format("Failed creating the hive metastore client for catalog: %s", name), e);
+                    String.format("Failed creating the hive metastore client for catalog: %s", catalogName), e);
         }
-        final Module hiveModule = new HiveConnectorModule(name, configuration, infoConverter, client);
+        final Module hiveModule = new HiveConnectorModule(catalogName, configuration, infoConverter, client);
         final Injector injector = Guice.createInjector(hiveModule);
         this.databaseService = injector.getInstance(ConnectorDatabaseService.class);
```

Comment:
While this is probably fine cause we know the inner workings of the module/injector and you're just going to return a singleton it's probably a bad practice in general because if the injector really returns a different instance every time with different logic (e.g. a provider method) then users wouldn't get new updated versions of the database service every time. They'd be stuck with the old one.